tests/integration/kubernetes/runtimeclass_workloads/pod-sealed-secret.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-test-pod-cc
+spec:
+  runtimeClassName: kata
+  containers:
+  - name: busybox
+    image: quay.io/prometheus/busybox:latest
+    imagePullPolicy: Always
+    command:
+    - sh
+    - -c
+    - |
+      env
+      if [ "$SECRET_PASSWORD" == "unsealed_secret" ]; then
+        echo "unsealed environment as expected"
+      fi
```

Comment:
Idea: I'm wondering whether having something like `echo "Unsealed env secret: $SECRET_PASSWORD"` and then including the secret content in the grep check in the test might be a bit more customisable?

Reply:
Good idea. Ideally we could generate a random key each time and then package it into a secret. I haven't gone that far, but I've move the check from the yaml file to the bash script.